scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -87,6 +87,34 @@ impl ReplicatorConsumer {
         self.update_or_insert(data, true).await;
     }
 
+    // Initializes basic data from CDC row.
```

Comment:
What is "basic data"? The comment is more confusing than helpful since it uses an undefined term, I need to read the function body to understand that "basic data" is TTL, write timestamp, and values for primary key columns.